test-it/ViaTests/TestVI_MultipleDelimiters.via

```diff
@@ -0,0 +1,116 @@
+// Autogenerated Vireo assembly file
```

Comment:
Was it your intention to add this test? it seems unrelated to the change and I don't see it in the testList.json file.

Reply:
It is in the testList.json file, look again.

Reply:
And it's not unrelated, Spreadsheet to Array now takes an array of delimiters.

Reply:
Sorry, my intention was to refer to OctalTest.via, this one is OK.